cmd/npmExecuteScripts_test.go

```diff
@@ -31,10 +31,23 @@ func TestNpmExecuteScripts(t *testing.T) {
 		utils.AddFile("package.json", []byte("{\"name\": \"Test\" }"))
 		utils.AddFile("src/package.json", []byte("{\"name\": \"Test\" }"))
 
-		npmExecutor := npm.NpmExecutorMock{Utils: utils, Config: npm.NpmConfig{Install: config.Install, RunScripts: config.RunScripts, PackagesList: config.BuildDescriptorList}}
+		npmExecutor := npm.NpmExecutorMock{Utils: utils}
 		err := runNpmExecuteScripts(&npmExecutor, &config)
 
-		assert.NoError(t, err)
+		if assert.NoError(t, err) {
+			assert.Equal(t, []string{"src/package.json"}, npmExecutor.Received.PackagesList)
+			if assert.Len(t, npmExecutor.Received.RunScripts, 2) {
+				assert.Subset(t, []string{"ci-build", "ci-test"}, npmExecutor.Received.RunScripts)
```

Comment:
a lot of these asserts are repeated in all but the last test. Maybe a function to run the common asserts would be more DRY.
Something in the line of `commonAsserts(t, npmExecutor.Received)`